airflow/utils/log/filename_formatter.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+
+def format(task_instance, **kwargs):
```

Comment:
Oh I thought you mean we need some other abstraction for the filename. This string approach sounds good to me.